### ISSUE TITLE
Both democracy and "authority" are types of authority

### ISSUE THREAD

**Issue by nissubjtsdjarwylwz** [8values/8values.github.io#128]
Hi, your democracy vs authortiy is biased towards "you either democrat or authoritarian". Both democracy and authority are types of authority which simply have a different government system. The opposite of authority is liberty, and democracy is just a type of authority, where the majority is the oppressor. **Antonim of authority is LIBERTY and lack of authority is not democracy but libertariatnism**. Democarcy is a TYPE of AUTHORITY that allowed slavery and executed hundred thousands of homosexuals. 

I as a libertarian was deemed as "authoritatian" because I was not supporting oppresive democractical minority-hating mob-rule, while my ideology and the definition of freedom and is definition of anti-authority. The democracy means oppression of minority and individuals are not happy if the stick they are being beaten with is called "stick of the people". 

Replace democracy with liberty.  You are either a democrat or were paid by Biden's team because this test is clearly a biaded, there is no way democracy can be oppsed to authority, the world ha defined the freedom axis hundred year ago and it is called Libertarianism.

**Comment by Genora51** [comment]
You make an interesting point and the Bakunin quote does not go unnoticed; it is true that in many cases libertarians take the stance that democracy as a notion is inherently oppressive. Indeed, the anarchist and libertarian ideologies which oppose the principles of democracy on this basis [that it is a tool of oppression and a barrier to liberty] should ideally score strongly on the liberty axis.

Unfortunately, it seems the test does not do a particularly good job of differentiating between opposition to democracy on libertarian grounds and opposition on authoritarian ones. I am open to suggestions on how the test could be practically expanded and improved in order to take into account the nuanced approach to democratic ideals taken by many libertarians and anarchists.

**Comment by ten7ei** [comment]
I don't agree completely here, however I might lack some deeper understanding of the ideals of anarchim or libertarianism. Democracy means that the population has the power to decide. The Democratic party has nothing to do with that. The system in the USA has many different problems which are antidemocratic because many things reduce the influence from the population. Ideal democracy is in my opinion anarchism or libertarianism. If there is no hierarchical governing body then the population has the power.

I see the problem of the power of the majority, but this is not inherent to democracy. Any form of dicision making from several people has this problem. Thats why decisions should be made on consensus or from each most negative affected party/stakeholder. But in the end still all people have to agree on it somehow if you don't want an authority to decide that.
That why "real" democracy is part of anarchism as far as can see. But I'm open to change my mind.
